### DISCUSSION TITLE
Surprised that MP3 available and worked

### DISCUSSION THREAD

**Discussion by Remmele** [aclap-dev/video-downloadhelper#1271] [Help and Q&A]
### Your question

I went to the website for radio 103.7 and my Beta 9.0.2.12 and to my surprise DownloadHelper gave me MP3 and recorded it with no problem.  Now it is only giving me MP4 or MP4A either RawHls or HTPPMedia.  Just wanted find out if the MP3 was just a fluke or what.

### Addon & CoApp version and details

![image](https://github.com/user-attachments/assets/fc52018b-e88c-4ba0-b0b7-b4fca98587c9)

**Comment by jcfrog** [comment]
hi,
if you click on the 3 little dots on the right of the button, you'll get a "Download Audio".

<img width="337" alt="image" src="https://github.com/user-attachments/assets/8f80abf5-07de-44fe-b36e-0f578623e1f7">

**Comment by Remmele** [comment]
Thanks, I don't know how it defaulted to it the first time.  I had only
recorded MP4 but found that station and just tried it, and it seemed to
automatically pick it up as MP3 but weeks later not, so thanks for the tip,
it worked fine for me.
